tutorial_2.md

```diff
@@ -53,158 +81,102 @@ git clone https://github.com/JackKuo666/semanticscholar-MCP-Server.git
 
-## Step 2: Setup Dependencies (2 minutes)
-
-Install the bootstrap system dependencies:
+## Step 2: Deploy with API Automation (2 minutes)
 
-```bash
-cd bootstrap
-uv pip install -r requirements.txt
-```
-
-The bootstrap system reads your `.env` file for the LaunchDarkly API key and creates everything programmatically using the LaunchDarkly CLI under the hood.
-
-## Step 3: Understand the Segmentation Strategy (3 minutes)
-
-Now that you have MCP tools installed, let's understand how they'll be distributed across different user segments.
-
-Your CLI will create 4 combined user segments for precise targeting:
-
-### Combined Segments (Geography + Business Tier)
-- **EU Free**: European users on free plans - get Claude Haiku with basic search only
-- **EU Paid**: European users on paid plans - get Claude Sonnet with full MCP research tools  
-- **Other Free**: Non-EU users on free plans - get GPT-4o Mini with basic search only
-- **Other Paid**: Non-EU users on paid plans - get GPT-4 with full MCP research tools
-
-### Simplified Targeting Matrix
-
-```
-                │  Free           │  Paid
-────────────────┼─────────────────┼─────────────────
-EU Users        │  Claude Haiku   │  Claude Sonnet
-                │  Basic Search   │  + Full MCP
-Other Users     │  GPT-4o Mini    │  GPT-4  
-                │  Basic Search   │  + Full MCP
-```
-
-**Why This Works:**
-- **Cost Optimization**: Free users get efficient models, Paid users get premium capabilities
-- **Simplified Management**: 4 segments instead of complex geographic × tier combinations
-- **Enhanced Privacy**: EU users get Anthropic Claude models with privacy-by-design approach
-
-## Step 4: Deploy with CLI Bootstrap (2 minutes)
-
-In [Part 1](README.md) you saw how easy it was to set up AI Configs through the [LaunchDarkly UI](https://app.launchdarkly.com), for this step we will use the Command Line Interface (CLI). [LaunchDarkly's CLI](https://docs.launchdarkly.com/home/getting-started/ldcli) lets you manage tools and AI Configs programmatically. Instead of manually creating dozens of variations, you'll deploy complex targeting matrices with a single command. This is essential when you need to handle multiple geographic regions × business tiers.
+Now we'll use programmatic API automation to deploy the complete configuration including tools and targeting matrix. The [LaunchDarkly REST API](https://launchdarkly.com/docs/guides/api/rest-api) lets you manage tools, segments, and [AI Configs](https://launchdarkly.com/docs/home/ai-configs) programmatically. Instead of manually creating dozens of variations in the UI, you'll deploy complex targeting matrices with a single Python script. This approach is essential when you need to handle multiple geographic regions × business tiers with conditional tool assignments.
 
 Deploy your complete targeting matrix with one command:
 
 ```bash
+cd bootstrap
 uv run python create_configs.py
```

Comment:
This was a magic moment. the terminal output is really nicely formatted! So much easier than doing this via the UI! Bootstrapping AI Configs changes via API could probably be a whole separate tutorial.